packages/router/package.json

```diff
@@ -4,11 +4,14 @@
   "files": [
     "dist"
   ],
-  "module": "dist/index.js",
+  "main": "dist/index.js",
   "license": "MIT",
   "dependencies": {
     "core-js": "3.6.4"
   },
+  "peerDependencies": {
```

Comment:
Another wowza —> https://nodejs.org/es/blog/npm/peer-dependencies/#the-solution-peer-dependencies

🤔